fns-hl7-pipeline/fn-mmg-validator/pom.xml

```diff
@@ -318,5 +323,15 @@
 
         </plugins>
     </build>
+	<distributionManagement>
```

Comment:
fix the spacing - make sure the distribution management lines up.  Also test this POM file update by checking out to local machine and running a build